0-RegEx.md

```diff
@@ -13,84 +13,84 @@ It's recommended that the student uses https://regex101.com/ when attempting to
 Provide three regexes that matches `yes yes yes` but not `no no no`
 
 ```c#
-answer:
+answer: \w\w\w\b\s\w\w\w\b\s\w\w\w\b     ^\w\w\w\b\s\w\w\w\b\s\w\w\w\b$       ^\w\w\w\s\w\w\w\s\w\w\w$
 
 ```
 
 Provide a regex that matches valid phone numbers with the forms `678-123-1122` and `(678) 123-1122` while still allowing symbols to be optional. HINT: Research the rules around valid phone numbers.
 
 ```c#
-answer:
+answer:^[(]?\d{3}[)]?-\d{3}-\d{4}
 
 ```
 
 Provide a regex that matches URLs like `https://regex101.com`, `http://facebook.com` and `https://www.godaddy.com` but not `https://msdn.microsoft.com`. (Trailing `/` should be allowed and optional)
 
 ```c#
-answer:
+answer:^\w+[:]\/+\w+.\w+\s
 
 ```
 
 Provide a regex that matches dates with the format: `Wed Aug 11, 2013`.
 
 ```c#
-answer:
+answer:^\w+\s\w+\s\d{2}[,]\s\d{4}
 
 ```
 
 Provide a regex that matches dates with the format: `MM/DD/YYYY`. Use named captures to extract Day, Month and Year.
 
 ```c#
-answer:
+answer:^(\w+)\/(\w+)\/(\w+)
 
 ```
 
 Provide a regex that matches `function returnOne() {return 1;}` and captures the value returned.
 
 ```c#
-answer:
+answer:^\w+\s\w+\(\)\s({\w+\s\d;})
 
 ```
 
 Provide a regex that matches `List<int> my_List = new List<int>();` and captures the variable name.
 
 ```c#
-answer:
+answer:^\w+<\w+>\s(\w+)\s=\s\w+\s\w+<\w+>[(][)];
 
 ```
 
 Provide two regeesx that matches `Billy Jean` but not `billy jean`.
 
 ```c#
-answer:
+answer:^[A-Z]\w+\s[A-Z]\w+
```

Comment:
this works - another one would be `(?<firstName>[A-Z]\w+)\s+(?<lastName>[A-Z]\w+)\s+`